src/precompiles/native.rs

```diff
@@ -0,0 +1,267 @@
+use evm::{Context, ExitError, ExitSucceed};
+
+use super::{Precompile, PrecompileResult};
+use crate::prelude::{String, ToString, Vec, U256};
+use crate::types::AccountId;
+
+mod costs {
+    use crate::types::Gas;
+
+    // TODO(#51): Determine the correct amount of gas
+    pub(super) const EXIT_TO_NEAR_GAS: Gas = 0;
+
+    // TODO(#51): Determine the correct amount of gas
+    pub(super) const EXIT_TO_ETHEREUM_GAS: Gas = 0;
+
+    // TODO(#51): Determine the correct amount of gas
+    pub(super) const FT_TRANSFER_GAS: Gas = 100_000_000_000_000;
+
+    // TODO(#51): Determine the correct amount of gas
+    pub(super) const WITHDRAWAL_GAS: Gas = 100_000_000_000_000;
+}
+
+/// Get the current nep141 token associated with the current erc20 token.
+/// This will fail is none is associated.
+fn get_nep141_from_erc20(_erc20_token: &[u8]) -> AccountId {
+    // TODO:
```

Comment:
Where it will TODO? If not planned now we should create an issue.

Reply:
it is done in my pr, didnt remove for it to compile